diff --git a/lib/core/device.js b/lib/core/device.js
index ffadc63f15e8b0b435e640af60c6b854d0236a83..8a09faebae021f845d961086c8bb2b9ee19096f3 100644
--- a/lib/core/device.js
+++ b/lib/core/device.js
@@ -105,9 +105,9 @@ var Platform;
 })(Platform = exports.Platform || (exports.Platform = {}));
 const mobile = {
     id: "com.tencent.mobileqq",
-    name: "A8.8.80.7400",
-    version: "8.8.80.7400",
-    ver: "8.8.80",
+    name: "A8.9.25.10005",
+    version: "A8.9.25.10005",
+    ver: "8.9.25",
     sign: Buffer.from([166, 183, 69, 191, 36, 162, 194, 119, 82, 119, 22, 246, 243, 110, 182, 141]),
     buildtime: 1640921786,
     appid: 16,